custodia/cli/__init__.py

```diff
@@ -14,6 +14,8 @@
 
 import pkg_resources
 
+
+from requests.exceptions import ConnectionError as ConnHTTPError
```

Comment:
Please don't rename the exception like that. ConnectionError is fine. Or name it RequestsConnectionError.